lib/aws.py

```diff
@@ -256,6 +257,29 @@ def instance_outdated_launchtemplate(instance_obj, asg_lt_name, asg_lt_version):
     return False
 
 
+def older_instance (instance_id, days_fresh):
```

Comment:
Since this function returns True or False I would call it `is_instance_outdated`

Reply:
I ended-up following the same convention they used like  `instance_outdated_launchconfiguration`, `instance_outdated_launchtemplate`. This one I called `instance_outdated_age`. 

Maybe that's better?